src/ParameterDefinition.php

```diff
@@ -45,15 +45,6 @@ public function isOptional(): bool
         return $this->parameter->isOptional();
     }
 
-    public function isBuiltin(): bool
-    {
-        $type = $this->parameter->getType();
-        if ($type === null) {
-            return false;
-        }
-        return $type->isBuiltin();
```

Comment:
Have you considered keeping this method?
```php
public function isBuiltin(): bool
    {
        $type = $this->parameter->getType();
        if ($type === null) {
            return false;
        }
        return !$this->isUnionType() && $type->isBuiltin();
}
```

Reply:
> Have you considered keeping this method?

No. For union type method `isBuiltin()` doesn't make sense. I think that's why it was removed from `ReflectionType`.